packages/base/package.json

```
@@ -33,10 +33,10 @@
     "test:unit:ie": "npm run test:unit:default -- --browsers=IE"
   },
   "dependencies": {
-    "@jupyterlab/services": "^6.0.0",
-    "@lumino/coreutils": "^1.11.1",
-    "@lumino/messaging": "^1.10.1",
-    "@lumino/widgets": "^1.30.0",
+    "@jupyterlab/services": "^7.0.0-alpha.19",
+    "@lumino/coreutils": "^2.0.0-beta.0",
+    "@lumino/messaging": "^2.0.0-beta.0",
+    "@lumino/widgets": "^2.0.0-beta.0",
```

I'd suggest being soft on the Lumino dependencies:

```suggestion
    "@lumino/coreutils": "^1.11.1 || ^2.0.0-beta.0",
    "@lumino/messaging": "^1.10.1 || ^2.0.0-beta.0",
    "@lumino/widgets": "^1.30.0 || ^2.0.0-beta.0",
```

That way people won't get warnings in the JS console with JupyterLab 3.

---

yes, that would be the goal. I'd like first to get the CI green with more strict constraints on lumino.